app/models/workplace.rb

```diff
@@ -0,0 +1,7 @@
+class Workplace < ApplicationRecord
+  belongs_to :employed_user, class_name: "User", foreign_key: :user_id
```

Comment:
@emcooper I like the distinction this creates. I imagine we now have the following relationships: 

@workplace.employed_user => User object ;  and @user.workplaces => [Workplace objects]